README.md

```diff
@@ -56,12 +56,24 @@ diesel 2.0.0
 
 # Set up
 
+All environment variables required are set the `.env` file
+
+## Start database
+
+Run
+
+```bash
+make db_start
+```
+
+This will create and launch a docker container running a postgres database running at `DATABASE_URL`
+
```

Comment:
Question: Should the readme have info on how to start the server?

Reply:
At a period after every sentence please.